zegami_sdk/util.py

```diff
@@ -108,12 +107,36 @@ def _auth_post(self, url, body, return_response=False, **kwargs):
 
 def _auth_put(self, url, body, return_response=False, **kwargs):
     """Syncronous PUT request. Used as standard over async currently.
-
     If return_response == True, the response object is returned rather than
     its .json() output.
-
     Any additional kwargs are forwarded onto the requests.put().
     """
     r = requests.put(url, body, headers=self.headers, **kwargs)
     self._check_status(r, is_async_request=False)
     return r if return_response else r.json()
+
+
+def _create_singed_blob_storage(self, workspace_id):
```

Comment:
Could we rename this to `_obtain_signed_blob_storage_url`?

Reply:
> Looks good. A few extra small points worth addressing.

Thanks, Andy, all points have been address in the latest commit.